src/components/Button/Button.tsx

```diff
@@ -1,54 +1,24 @@
 import React from 'react';
 import styles from './Button.module.css';
 
-interface ButtonProps {
-  /**
-   * Is this the principal call to action on the page?
-   */
-  primary?: boolean;
-  /**
-   * What background color to use
-   */
-  backgroundColor?: string;
-  /**
-   * How large should the button be?
-   */
-  size?: 'small' | 'medium' | 'large';
-  /**
-   * Button contents
-   */
-  label: string;
-  /**
-   * Optional click handler
-   */
-  onClick?: () => void;
-}
+type ButtonElementProps = {
+  text: string;
```

Comment:
You could remove this and use the children prop instead, that way you could technically also have something else than a string as content for the button